README.md

```diff
@@ -1 +1 @@
-# gymkhana-sac
\ No newline at end of file
+# Student Activity Council IITJ
```

Comment:
Also add steps on how to run the project locally.

Reply:
Done